app.yaml

```diff
@@ -0,0 +1,25 @@
+#application: tutorial-trial
+#version: 1
+runtime: python27
+api_version: 1
+threadsafe: yes
+
+handlers:
+- url: /_ah/push-handlers/.*
+  script: main.app
+  login: admin
+
+- url: /stylesheets
+  static_dir: stylesheets
+
+- url: /static
+  static_dir: static
+
+- url: .*
+  script: main.app
+
+libraries:
```

Comment:
If you're using vendoring, don't depend on these libraries. Just add them to your requirements.txt.